tests/common/plugins/pdu_controller/pdu_manager.py

```diff
@@ -50,8 +50,8 @@ def __init__(self, dut_hostname):
         self.controllers = []
 
     def _update_outlets(self, outlets, pdu_index, controller_index=None):
-        for outlet_idx, outlet in enumerate(outlets):
```

Comment:
The original code is has below commit message
"Fix the bug while two PSU port connect to the same PDU server".
Does the new code cover that scenario?